tests/vtep/tests.c

```diff
@@ -1099,93 +1096,169 @@ static void test_vtep_flood_encapsulate(void)
 	pg_packets_free(pkts, pg_mask_firsts(NB_PKTS));
 }
 
-static void test_vtep_fragment_encap_decap(void)
+static void free_collectors(struct pg_brick *(*collects_ptr)[20])
 {
+	struct pg_brick **collectors = *collects_ptr;
+
+	for (int i = 0; i < NB_VNIS; ++i)
+		pg_brick_destroy(collectors[i]);
+}
+
+static void test_vtep_lifetime(void)
+{
+	struct ether_addr mac_src = {{0xb0, 0xb1, 0xb2,
+				       0xb3, 0xb4, 0xb5}};
 	struct pg_error *error = NULL;
-	struct pg_brick *ip_fragment;
-	struct pg_brick *vtep;
-	struct pg_brick *collect_west;
-	struct pg_brick *collect_east;
+	struct ether_addr mac[] = {
+		{{0xc0, 0xc1, 0xc2, 0xc3, 0xc4, 0xc0}},
```

Comment:
Just a question... Isn't it not better to make a func generating this table? Or would it be worst? (I'm not sure...)